spec/features/peep_view_spec.rb

```diff
@@ -0,0 +1,74 @@
+describe 'chitter' do
+  feature 'View peeps' do
+    scenario 'User can view peeps' do
+      Peep.add(peep: 'Makers', account: 'Makers')
+      visit('/')
+      expect(page).to have_content('Makers')
+    end
+  end
+
+  feature 'Post peeps' do
+    scenario 'User can post a peep to chitter' do
+      con = PG.connect(dbname: 'chitter_test')
+      visit('/')
+      fill_in('peep', with: 'This is a new peep')
+      click_button('New Peep')
+      expect(page).to have_content('This is a new peep')
+    end
+  end
+
+feature 'Delete peeps' do
+  scenario 'User can delete peeps' do
+    con = PG.connect(dbname: 'chitter_test')
+    visit('/')
+    fill_in('peep', with: 'This is a new peep')
+    click_button('New Peep')
+    expect(page).to have_content('This is a new peep')
+    first('.peep').click_button 'Delete'
+    expect(current_path).to eq '/'
+    expect(page).not_to have_content('This is a new peep')
+  end
+end
+
+feature 'User Sign Up' do
+  scenario 'User can sign up to chitter' do
+    con = PG.connect(dbname: 'chitter_test')
+    visit('/')
+    click_button('Sign up for Chitter here')
+    expect(current_path).to eq '/users/new_user'
+    fill_in('name', with: 'Lisa')
+    fill_in('email', with: 'lisa@thesimpsons.com')
+    fill_in('password', with: 'springfield')
+    click_button('Sign Up')
+    expect(page).to have_content('Welcome Lisa')
+  end
+end
+
+feature 'Edit Peeps' do
+  scenario "User can edit a posted peep message" do
+    peep = Peep.add(peep: 'Makers is great', account: 'Makers')
+    visit('/')
+    expect(page).to have_content('Makers is great')
+    first('.peep').click_button 'Edit'
+    expect(current_path).to eq "/#{peep.id}/edit"
+    fill_in('peep', with: "Makers is the best!")
+    click_button('Edit Peep')
+    expect(current_path).to eq '/'
```

Comment:
Your tests look really comprehensive and especially like the expect(current_path) tests - I will do this more next time!